src/components/NavMenu/NavMenu.tsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from "react-router-dom";
+
+import Styled from "./NavMenu.styled";
+
+const navLinks = [
+  {
+    to: "/",
+    title: "Home",
+  },
+  {
+    to: "/breeds",
+    title: "Breeds",
+  },
+  {
+    to: "/favourites",
+    title: "Favourites",
+  },
+] as const;
```

Comment:
Why have you used `as const` here?

Reply:
Using `as const` here solves several problems:

1. It prevents us or other developers from mutating any key/values in the navLinks array in the future.
2. We don't need a separate type definition for this array.